demo/GDI/renderer.c

```diff
@@ -8,9 +8,15 @@
 static int width = 800;
 static int height = 600;
 
-int running;
 static int needs_refresh = 1;
 
+
+#ifdef __MINGW32__
```

Comment:
It should be
```c
#ifndef __MINGW32__
       int running;
#endif
```
I've not yet tested it under mingw, but is there any issue while using mingw?

Reply:
I'll be honest I cant remember - its been a while since I have looked at this. I do think there was a issue with exiting the window though yes.